src/src/hooks/Capabilities.js

```diff
@@ -290,3 +290,53 @@ export function useCapabilityMembersApplications(capabilityDefinition) {
     membersApplicationsList,
   };
 }
+
+export function useCapabilityMetadata(capabilityDefinition) {
+  const { responseData, sendRequest: sendGetJsonMetadataRequest } =
+    useSelfServiceRequest();
+  const { sendRequest: sendSetJsonMetadataRequest } = useSelfServiceRequest();
+  const [isLoadedMetadata, setIsLoadedMetadata] = useState(false);
+  const [metadata, setMetadata] = useState(null);
+
+  const link = capabilityDefinition?._links?.metadata;
+
+  useEffect(() => {
+    if (link && (link.allow || []).includes("PUT")) {
```

Comment:
Are you sure this should not be a `GET`.
I believe this would work, as I don't think we currently ever have a `PUT` without a `GET`.
Regardless, it should probably be changed

Reply:
addressed